k8s-scheduler/src/test/java/com/vmware/dcm/ScopeTest.java

```diff
@@ -190,6 +253,124 @@ public void testScopedSchedulerNodeLabels() {
                 || ssdNodesIdx.contains(Integer.parseInt(e.getNodeName().substring(1))))));
     }
 
+    /*
+     * E2E test with scheduler:
+     * Test if Scope does not filter out nodes required by inter-pod affinity.
+     *
+     * Pods in group 0 are affine to each other.
+     * Pods in group 1 are affine to an already running pod.
+     * Pods in group 2 are anti-affine to each other and to a running pod.
+     */
+    @Test
+    public void testVariousPodAffinities() {
```

Comment:
Can you provide a sharper name than test "Various" ? How about "testPodAffinityMixed"?